examples/README.md

```diff
@@ -0,0 +1,48 @@
+# Merlin Models Example Notebooks
+
+We have created a collection of Jupyter notebooks based on different datasets. These example notebooks demonstrate how to use Merlin Models with TensorFlow.
```

Comment:
"We" is the pattern in notebooks, but isn't particularly effective in READMEs.  Maybe:

"The example notebooks demonstrate how to use Merlin Models with TensorFlow on a variety of datasets."